CONTRIBUTING.md

```diff
@@ -0,0 +1,91 @@
+# How to contribute
+
+There are lots of ways to contribute to the Uno Platform, and we appreciate the help from the community. You can provide feedback, report bugs, give suggestions, contribute code, and participate in the platform discussions.
+
+For a full list of topics, visit our [documentation for contributors](https://platform.uno/docs/articles/uno-development/contributing-intro.html).
+
+Contribute to Uno in your browser using [GitPod.io](https://gitpod.io): [![Open Uno in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/unoplatform/uno).
+
+## Code of conduct
+
+To better foster an open, innovative, and inclusive community, please refer to our [Code of Conduct](CODE_OF_CONDUCT.md) when contributing.
+
+## Provide feedback
+
+The Uno Platform is an ongoing effort, and as Microsoft progresses on WinUI, the Uno Platform follows the trails. While the development remained closed for a while, the areas covered by Uno may not suit everyone's needs, which is why your feedback is important to us.
+
+We want to hear about your experience, scenarios, and requirements.
+
+### Report a bug
+
+If you think you've found a bug, the first thing to do is [search the existing issues](https://github.com/unoplatform/Uno/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Fbug) to check if it's already been reported.
+
+If not, please [log a new bug report](https://github.com/unoplatform/uno/issues/new?labels=kind%2Fbug%2C+triage%2Funtriaged&template=bug-report.md).
+
+The best way to get your bug fixed is to be as detailed as you can be about the problem.
+Providing a minimal project with steps to reproduce the problem is ideal.
+Here are questions you can answer before you file a bug to make sure you're not missing any important information.
+
+1. Did you read the [documentation](https://platform.uno/docs/articles/intro.html)?
+2. Did you include the snippet of the broken code in the issue?
+3. What are the *EXACT* steps to reproduce this problem?
+4. What specific version or build are you using?
+5. What operating system are you using?
+6. What platform(s) are you targeting?
+
+GitHub supports [Markdown](https://help.github.com/articles/github-flavored-markdown/), so when filing issues, be sure to check the formatting in the 'Preview' tab before hitting submit.
+
+### Request a feature
+
+If you need a [WinUI feature](https://learn.microsoft.com/uwp/api/microsoft.ui.xaml.controls) that Uno doesn't support yet, you should [submit a feature request](https://github.com/unoplatform/uno/issues/new?labels=kind%2Fenhancement%2C+triage%2Funtriaged&template=enhancement.md). Check [existing issues first](https://github.com/unoplatform/uno/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Fenhancement) in case the same feature request already exists (in which case you can upvote the existing issue).
```

Comment:
Let's use Uno Platform wherever possible

```suggestion
If you need a [WinUI feature](https://learn.microsoft.com/uwp/api/microsoft.ui.xaml.controls) that Uno Platform doesn't support yet, you should [submit a feature request](https://github.com/unoplatform/uno/issues/new?labels=kind%2Fenhancement%2C+triage%2Funtriaged&template=enhancement.md). Check [existing issues first](https://github.com/unoplatform/uno/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Fenhancement) in case the same feature request already exists (in which case you can upvote the existing issue).
```